keps/sig-cli/0031-kustomize-integration.md

```diff
@@ -199,35 +338,50 @@ Most implementation will be in cli-runtime
 - [x] vendor `kustomize/pkg` into kubernetes
 - [x] copy `kustomize/k8sdeps` into cli-runtime
 - [x] Implement a Visitor for kustomization directory which
```

Comment:
We don't need to implement a visitor to kustomization directory since the subcommand `kustomize` we're adding doesn't rely on builder or visitor interface.